data/data/config.tf

```diff
@@ -61,7 +61,7 @@ EOF
 
 }
 
-variable "ignition_bootstrap" {
+variable "ignition_worker" {
```

Comment:
can you explain why this is needed? I think workers should generally pull their config from the machine-config operator

Reply:
they do, just like master. They start with stub ignition config that points to the machine-config-server and once its up they pull it.

Reply:
the workers should not be created by the terraform at all right? because that is taken care of by the machine-api